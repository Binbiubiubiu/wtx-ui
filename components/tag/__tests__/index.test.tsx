import React from 'react';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';
import Tag from '..';

describe('<Tag />', () => {
  it('should render a <Tag/> components', () => {
    const wrapper = render(
      <div>
        <Tag>默认</Tag>
        <Tag radius>默认</Tag>
      </div>,
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
