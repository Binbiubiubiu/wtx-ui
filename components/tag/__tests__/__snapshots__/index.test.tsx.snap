// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tag /> should render a <Tag/> components 1`] = `
<div>
  <div
    class="wtx-tag"
  >
    默认
  </div>
  <div
    class="wtx-tag wtx-tag__radius"
  >
    默认
  </div>
</div>
`;
